src/main/java/org/candlepin/model/AbstractHibernateCurator.java

```diff
@@ -915,8 +915,10 @@ protected E lockAndLoad(Class<E> entityClass, Serializable id) {
     }
 
     /**
-     * Loads the entities represented by the given IDs with a pessimistic write lock. If no
-     * entities were found with the given IDs, this method returns an empty collection.
+     * Loads the entities represented by the given IDs with a pessimistic write lock. If no entities
+     * could be found matching the given IDs, this method returns an empty collection. Note that this
+     * method makes no attempt to ensure that an entity is loaded for every ID provided. It is possible
```

Comment:
Can you explain what you mean by `makes no attempt to ensure that an entity is loaded for every ID provided` ? Do you mean it in the same way as the 'Note' below? If so, isn't that redundant?